src/redux/auth/reducer.ts

```diff
@@ -11,30 +11,27 @@ import {
   LOGIN_WITH_GOOGLE,
   LOGOUT,
   RequestStatus,
-  SET_DID_TRY_AUTO_LOGIN,
   SIGNUP,
   SIGNUP_FAILURE,
   SIGNUP_SUCCESS,
   TRY_AUTO_LOGIN,
 } from './types';
 
 export interface AuthState {
-  token: string | null;
+  isAuthenticated: boolean | null;
```

Comment:
The question is, if isAuthenticated should be equal false initially? I would prefer rather null, to have knowledge whether auth state has checked. What you think?

Reply:
Sure, I won't force it, we can have it as `boolean | null` with no problem :+1: 

I just put my arguments:
* usually when I've got `isSomething` variable I expect it to be only `true` or `false`,
* previously we should have some computation logic for `isAuthenticated` (probably `!!state.token`). So in case of empty token it evaluated to `false` and in other cases `true` - no `null` here :smile:

And that's it :smile: